packages/jellyfish-transaction/src/script/crypto.ts

```diff
@@ -18,6 +18,15 @@ export class OP_SHA1 extends StaticCode {
   }
 }
 
+/**
+ * The input is hashed using SHA-3.
+ */
+export class OP_SHA3 extends StaticCode {
+  constructor () {
+    super(0xc0, 'OP_SHA3')
```

Comment:
Just checking doesn't this cause any collision with existing OP_CODE from Bitcoin Core.

Reply:
Don’t think so, it’s not used at all by bitcoin